test/integration/2-stackstorm/inspec.yml

```diff
@@ -0,0 +1,14 @@
+name: stackstorm
+title: Integration Tests for StackStorm community
+summary: >
+  Tests to verify if StackStorm packages st2, st2web, st2mistral, st2chatops are installed,
+  files and directories are shipped, services are running as well as dependent servers like
+  nginx, postgresql, mongodb are available and working correctly.
```

Comment:
BTW @nmaludy you might be interested in taking these tests for `puppet-st2` repo when adding InSpec there.

We just moved these controls from the private repo.